JavaTask/MultiThread.java

```diff
@@ -0,0 +1,75 @@
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
+class FileConcept extends Thread
+{
+    public void run()
+    {
+    try {
+    File file = new File("read.txt");
```

Comment:
dont simply put the entire code in run method instead create thread instance and copy files with the help of thread instance.